### DISCUSSION TITLE
Alternative to Container.worldVisible in v8?

### DISCUSSION THREAD

**Discussion by sam007mac** [pixijs/pixijs#10105] [v8 General]
Looking though the docs I can't find a way to see if a Container is visible based on parent visibility. (side note, https://pixijs.download/v8.0.0-rc.4/docs/index.html docs have an Access Denied)

I had thought `Container.isRenderable` would do it but that also doesn't check for the visibility of the parents.

**Comment by bilasyurii** [comment]
I haven't tested it yet, but you could try something like this:


    public static worldVisible(object: Container): boolean {
        return (object.globalDisplayStatus & 0b10) !== 0;
    }
